Cargo.toml

```diff
@@ -28,7 +28,11 @@ regex = "1"
 lazy_static = "1.4"
 typed-arena = "2.0.2"
 reqwest = { version = "0.11.12", features = ["blocking"] }
+colored = "2.0.4"
+uuid = { version = "1.4.1", features = ["v4"] }
 
 [dev-dependencies]
 criterion = { version = "0.4", features = ["html_reports"] }
 test-case = "3.2.1"
+colored = "2.0.4"
+uuid = { version = "1.4.1", features = ["v4"] }
```

Comment:
Did you want the new libraries in both the regular and dev dependencies?  I think they would go in one section or the other, but not both.